tdei-api-gateway.json

```diff
@@ -243,121 +335,59 @@
                 ]
             }
         },
-        "/api/v1/gtfs-pathways": {
-            "get": {
+        "/api/v1/osw/publish/{tdei_record_id}": {
+            "post": {
                 "tags": [
-                    "GTFS-Pathways"
+                    "OSW"
                 ],
-                "summary": "List pathways files meeting criteria.",
-                "description": "This endpoint returns a json list of all gtfs pathways files stored in the TDEI system that meet the specified criteria. Criteria that can be specified include: polygon (bounding box), minimum confidence level, pathways version, date time and agency id.\n\nThis endpoint can be used by an application developer to obtain a list of gtfs pathways files in the TDEI system meeting the specified criteria. This endpoint returns a list of file-metadata including the uris of the file, which can be used to fetch the files themselves.",
-                "operationId": "listPathwaysFiles",
+                "summary": "Publishes the OSW dataset for the tdei_record_id",
+                "description": "Publishes an OSW dataset that has undergone prior uploading and editing via the [POST] /osw endpoint, marking it as an official release for the uploaded mobility service. This official release status ensures visibility to all TDEI data consumers.",
```

Comment:
Publishes an OSW dataset that was previously uploaded via the [POST] /osw endpoint, marking it as an official release for the mobility service. This official release status ensures visibility to all TDEI data consumers.

Reply:
Done